integration_test/cases/backoff_test.exs

```diff
@@ -16,12 +28,19 @@ defmodule BackoffTest do
         {:ok, :state}
       end]
     {:ok, agent} = A.start_link(stack)
+    agent
+  end
 
-    opts = [agent: agent, parent: self(), backoff_min: 10]
+  defp execute_test_backoff_after_failed(agent, opts, test_hibernate? \\ false) do
```

Comment:
If theres enough logic to share a function we could try to simplify the test when hibenating and not share function. We might have overlapping tests.